dev/ci/ci-user-overlay.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Add user overlays here. You can use some logic to detect if you are
+# in your travis branch and conditionally enable the overlay.
```

Comment:
Is there a point to conditionally enable the overlay since the overlay commit should be dropped before merge anyway?